geometry_msgs/Pose desired_bMe
---
float32[] out_joints
geometry_msgs/Pose bMe
bool found